Sources/Defaults/Observation.swift

```diff
@@ -304,6 +323,19 @@ extension Defaults {
 		return observation
 	}
 
+	public static func observe<Value: Serializable & Codable>(
```

Comment:
I don't think `observe` should need to know about the extra `Codable` logic. It feels weird having to add overloads for methods like this. The logic should be internal in the serialization.